CONTRIBUTING.md

```diff
@@ -40,3 +40,9 @@ The labels I assign to issues are:
 * **unconfirmed**: More information is needed to be sure this bug is really a bug.
 * **unlikely**: An enhancement or other change that I consider lowest priority, too large or difficult, or not appropriate.
 * **wontfix**: A change that I definitely do not think should be made.
+
+Some additional labels might be used to specify the scope of issues:
```

Comment:
```suggestion
```

There's no need to have these in a separate section. There's nothing separating "art" or "sound" from "content" or "enhancement" as an example.
I'd recommend putting them all in the same section.